File: Pangram1.vb
Module Module1

    Sub Main()
        Dim str1, alphaStr As String
        Dim nextChar As Char
        Dim Count As Integer
        Dim isPangram As Boolean

        str1 = ""
        alphaStr = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
        nextChar = ""
        Count = 0
        isPangram = True

        Console.Write("Enter String: ")
        str1 = Console.ReadLine
        str1 = UCase(str1)

        For Count = 1 To Len(alphaStr)
            nextChar = Mid(alphaStr, Count, 1)
            If InStr(str1, nextChar) = 0 Then
                isPangram = False
                Exit For
            End If
        Next

        If isPangram = True Then
            Console.WriteLine("Sentense enetred is pangram.")
        Else
            Console.WriteLine("Sentense enetred is NOT pangram.")
        End If
        Console.ReadKey()
    End Sub

End Module

File: Pangram 1/pangram1.py
alphaStr = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
isPangram = True

str1 = input("Enter String: ")
str1 = str1.upper()

for Count in alphaStr:
    if Count not in str1:
        isPangram = False
        break

if isPangram == True:
    print("Sentence entered is Pangram.")
else:
    print("Sentence Entered is NOT pangram.")